ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<b>OS:</b>
+*Override this line with the exact operating system you are running! (e.g. "Win 7 SP1", "OS X 10.8.5", "Ubuntu 15.10" ...)*
+
+<b>Cockatrice version:</b>
+*Put your Cockatrice version number here! You can find it inside the app: `Help` --> `About Cockatrice` (e.g. "2d53ce9")*
```

Comment:
Good point! :+1: 

Well, still not possible to copy text there on windows, see https://github.com/Cockatrice/Cockatrice/issues/1762